pymc3/stats.py

```diff
@@ -324,16 +325,20 @@ def loo(trace, model=None, pointwise=False, progressbar=False):
 
 
 def bpic(trace, model=None):
-    """
-    Calculates Bayesian predictive information criterion n of the samples in trace from model
+    R"""Calculates Bayesian predictive information criterion n of the samples in trace from model
     Read more theory here - in a paper by some of the leading authorities on Model Selection -
-    dx.doi.org/10.1111/1467-9868.00353
+    dx.doi.org/10.1080/01966324.2011.10737798
```

Comment:
Not familiar with either of these references -- is BPIC not in the old reference?  (if you get a chance to change it, I'd also not capitalize "**M**odel **S**election")

Reply:
The original paper has DIC. I linked the author of BPIC.

Ok